--- lycklama.c
+++ lycklama.c
@@ -1,16 +1,16 @@
 #define o define
-#o ___o write
-#o ooo (unsigned)
-#o o_o_ 1
-#o _o_ char
-#o _oo goto
-#o _oo_ read
-#o o_o for
-#o o_ main
-#o o__ if
-#o oo_ 0
-#o _o(_,__,___)(void)___o(_,__,ooo(___))
-#o __o (o_o_<<((o_o_<<(o_o_<<o_o_))+(o_o_<<o_o_)))+(o_o_<<(o_o_<<(o_o_<<o_o_)))
+#define ___o write
+#define ooo (unsigned)
+#define o_o_ 1
+#define _o_ char
+#define _oo goto
+#define _oo_ read
+#define o_o for
+#define o_ main
+#define o__ if
+#define oo_ 0
+#define _o(_,__,___)usleep(10000),(void)___o(_,__,ooo(___))
+#define __o (o_o_<<((o_o_<<(o_o_<<o_o_))+(o_o_<<o_o_)))+(o_o_<<(o_o_<<(o_o_<<o_o_)))
 o_(){_o_ _=oo_,__,___,____[__o];_oo ______;_____:___=__o-o_o_; _______:
 _o(o_o_,____,__=(_-o_o_<___?_-o_o_:___));o_o(;__;_o(o_o_,"\b",o_o_),__--);
 _o(o_o_," ",o_o_);o__(--___)_oo _______;_o(o_o_,"\n",o_o_);______:o__(_=_oo_(
