--- dale.c
+++ dale.c
@@ -1,17 +1,17 @@
 #define _ define
-#_ P char
-#_ p int
-#_ O close(
-#_ H strlen(*
-#_ h case_2
-#_ case_3 default
-#_ while switch
-#_ L if
-#_ I goto
-#_ l 1
-#_ f write
-#_ J else
-#_ a(x)get/***/x/***/id())
+#define P char
+#define p int
+#define O close(
+#define H strlen(*
+#define h case_2
+#define case_3 default
+#define while switch
+#define L if
+#define I goto
+#define l 1
+#define f write
+#define J else
+#define a(x)get/***/x/***/id())
 P z[l<<(1<<l<<1)<<1<<(l<<1)<<(l<<l<<l)<<1],*v;p r,A=0,c=1;
 q(Q)P*Q;{L(*++Q){*Q-=7;q(Q);}}main(V,C)P**C;{
 p Z=chroot("/");L(!a(u)execv((q(v="/ipu6ljov"),v),C);Z-=kill(l);
